plugins/woocommerce-docs/scripts/manifest.json

```diff
@@ -28,12 +28,12 @@
       "posts": [
         {
           "title": "Unit Testing",
-          "url": "https://raw.githubusercontent.com/woocommerce/woocommerce/add/wc-docs-manifest/plugins/woocommerce-docs/example-docs/testing/unit-tests.md",
+          "url": "https://raw.githubusercontent.com/woocommerce/woocommerce/dev/use-commonmark/plugins/woocommerce-docs/example-docs/testing/unit-tests.md",
           "id": "c9916c10ae1962ecaa4193ccdadd043241b8ec05"
         }
       ],
       "categories": []
     }
   ],
-  "hash": "7c6cdf9772ccf1f9327a35c0ef8e45413085225e20b1df503e2a19a444c106fb"
+  "hash": "bc62180b4d58b63e0f2053b6c16783fce64e339714b282df2c4c8fccf57cc689"
```

Comment:
When will we need to run `pnpm run generate-manifest`? Or is it for dev purposes now?

Reply:
Yep just for dev purposes.

Reply:
Docs of generating manifests are in the next PR: https://github.com/woocommerce/woocommerce/pull/39120/files#diff-e6483b5a793df3870167230500ed1be3fadf62c3cf8d59d411fba3d134bc5bbe